Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/SingleCreditCardView.swift

```diff
@@ -0,0 +1,306 @@
+        guard let hostingCell = cardTableView.dequeueReusableCell(
+            withIdentifier: HostingTableViewCell<CreditCardItemRow>.cellIdentifier) as? HostingTableViewCell<CreditCardItemRow> else {
+            return UITableViewCell(style: .default, reuseIdentifier: "ClientCell")
+        }
+
+        let creditCard = viewModel.creditCard
+        let creditCardRow = CreditCardItemRow(
+            item: creditCard,
+            isAccessibilityCategory: UIApplication.shared.preferredContentSizeCategory.isAccessibilityCategory)
+        hostingCell.host(creditCardRow, parentController: self)
+        hostingCell.isAccessibilityElement = true
+        return hostingCell
+    }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        guard let headerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: SingleCreditCardHeaderView.cellIdentifier) as? SingleCreditCardHeaderView else { return nil }
+        headerView.applyTheme(theme: themeManager.currentTheme)
+        headerView.viewModel = viewModel
+        return headerView
+    }
+
+    // easier to handle the bottom table spacing as a footer
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        switch viewModel.state {
+        case .save:
+            let emptyView = UIView()
+            return emptyView
+        case .update:
+            guard let footerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: SingleCreditCardFooterView.cellIdentifier) as? SingleCreditCardFooterView else { return nil }
+            footerView.applyTheme(theme: themeManager.currentTheme)
+            if !footerView.manageCardsButton.responds(to: #selector(SingleCreditCardViewController.didTapManageCards)) {
+                footerView.manageCardsButton.addTarget(self, action: #selector(SingleCreditCardViewController.didTapManageCards), for: .touchUpInside)
+            }
+
+            return footerView
+        }
+    }
+
+    func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        return UX.distanceBetweenButtonAndTable
+    }
+
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return UITableView.automaticDimension
+    }
+
+    // MARK: Themable
+    func applyTheme() {
+        let currentTheme = themeManager.currentTheme
+        let labelsBackgroundColor = currentTheme.type == .dark ? currentTheme.colors.textInverted : currentTheme.colors.textPrimary
+        view.backgroundColor = currentTheme.colors.layer1
+        contentView.backgroundColor = currentTheme.colors.layer1
+        yesButton.backgroundColor = currentTheme.colors.actionPrimary
+        yesButton.titleLabel?.textColor = labelsBackgroundColor
+
+        notNowButton.backgroundColor = currentTheme.colors.actionSecondary
+        notNowButton.titleLabel?.textColor = labelsBackgroundColor
```

Comment:
We shouldn't be checking for the theme to decide which color to use. In this case `layer5` is the correct color.
```suggestion
        view.backgroundColor = themeManager.currentTheme.colors.layer1
        contentView.backgroundColor = themeManager.currentTheme.colors.layer1
        yesButton.backgroundColor = themeManager.currentTheme.colors.actionPrimary
        yesButton.titleLabel?.textColor = themeManager.currentTheme.colors.layer5

        notNowButton.backgroundColor = themeManager.currentTheme.colors.actionSecondary
        notNowButton.titleLabel?.textColor = themeManager.currentTheme.colors.layer5
```

Reply:
the design has 2 different colors for the same button, in the theming part https://www.figma.com/file/cSKRUEvfJSzNBv2Mp65Pdv/Mobile-CC-Autofill?type=design&node-id=3060-168400&t=czlsDaVZaS7MFiVw-0